config/subsidy-application-management/forms/GZG/proposal/versions/20220524130000-sixth-call/form.ttl

```diff
@@ -0,0 +1,367 @@
+
+##########################################################
+# Did collaboration?
+##########################################################
+
+fields:abd39f3a-34ab-4898-9e3a-3c75dbfd9a1f a form:Field ;
+    mu:uuid "abd39f3a-34ab-4898-9e3a-3c75dbfd9a1f";
+    sh:name "Betreft deze subsidieaanvraag een aanvraag voor een samenwerkingsverband tussen gemeenten?" ;
+    sh:order 30 ;
+    sh:path lblodSubsidie:isCollaboration ;
+    form:validations
+        [ a form:ConceptSchemeConstraint ;
+            form:grouping form:Bag ;
+            sh:path lblodSubsidie:isCollaboration ;
+            form:conceptScheme <http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2> ;
+            sh:resultMessage "Selecteer een waarde uit de lijst."@nl
+        ],
+        [ a form:RequiredConstraint ;
+            form:grouping form:Bag ;
+            sh:resultMessage "Dit veld is verplicht."@nl;
+            sh:path lblodSubsidie:isCollaboration
+        ] ;
+    form:options  """{"conceptScheme":"http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2"}""" ;
+    form:displayType displayTypes:conceptSchemeRadioButtons ;
+    form:help """Bij een samenwerkingsverband dient één gemeente de aanvraag in.""" ;
+    sh:group fields:84baa661-ae79-4531-becc-b8636a928103 .
+
+fields:abd39f3a-34ab-4898-9e3a-3c75dbfd9a1f form:hasConditionalFieldGroup fields:3e03a32d-761f-479f-ace4-1f759e1c166f.
+
+fields:3e03a32d-761f-479f-ace4-1f759e1c166f a form:ConditionalFieldGroup ;
+    mu:uuid "3e03a32d-761f-479f-ace4-1f759e1c166f";
+    form:conditions
+        [ a form:SingleCodelistValue ;
+            form:grouping form:Bag ;
+            sh:path lblodSubsidie:isCollaboration ;
+            form:conceptScheme <http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2> ;
+            form:customValue <http://lblod.data.gift/concepts/9ffe7dd4-42b7-4af3-bdba-ccd080195715>
+        ] ;
+    form:hasFieldGroup fieldGroups:1c9b3ebf-549c-49ed-b06a-1c13d8e05062 .
+
+fieldGroups:1c9b3ebf-549c-49ed-b06a-1c13d8e05062 a form:FieldGroup ;
+    mu:uuid "1c9b3ebf-549c-49ed-b06a-1c13d8e05062" ;
+    form:hasField
+        ### Select collaboration
+        fields:8fa4216d-97b3-467a-bacc-c76b7ea6a242 .
+
+##########################################################
+# Collaboration
+##########################################################
+
+fields:8fa4216d-97b3-467a-bacc-c76b7ea6a242 a form:Field ;
+    mu:uuid "8fa4216d-97b3-467a-bacc-c76b7ea6a242" ;
+    sh:name "Welke gemeenten of intergemeentelijke samenwerkingsverbanden maken deel uit van dit samenwerkingsverband?" ;
+    sh:order  31;
+    sh:path lblodSubsidie:collaborator ;
+    form:validations
+        [ a form:Codelist ;
+            form:grouping form:MatchEvery ;
+            sh:path lblodSubsidie:collaborator ;
+            form:conceptScheme <http://lblod.data.gift/concept-schemes/7db0bcc4-7ccf-4848-affd-cbd41124ef71> ;
```

Comment:
This is not correct it should be https://github.com/lblod/app-digitaal-loket/blob/master/config/subsidy-application-management/forms/GZG/proposal/versions/20210721111449-first-call/form.ttl#L177